src/app/product-list/product-routing.module.ts

```diff
@@ -0,0 +1,15 @@
+import { ProductComponent } from './product/product.component';
```

Comment:
One of the very commonly followed module folder structure is
- [module_name]
- -components/
- -services
- module_name.routing.ts
- module_name.module.ts
Neither `-list` or `-routing` should be a part of the filenames if we follow that style

Reply:
Apply the same concept to other modules too